.travis.yml

```diff
@@ -7,6 +7,11 @@ os: osx
 osx_image: beta-xcode6.1
 
 env:
+  matrix:
+    
+    - CONDA_PY=27
+    - CONDA_PY=34
+    - CONDA_PY=35
```

Comment:
Oddly one can see that some Python versions did get included anyways. Still better safe than sorry to pull them and rebuild with this matrix intact.